### PR TITLE
Enabling Chromebook (R13) to download app on Play Store

### LINKED ISSUE #473: Acer Chromebook R13 not able to install through Google Play
When trying to install the demo app through Google Play on a Acer Chromebook R13.

I'm seeing:

 - `Your device isn't compatible with this version`

### PR DESCRIPTION
**Pull request by langsmith**
Fixes #473

### CODE REVIEW

**Comment by langsmith** [comment]
cc @tobrun

File: MapboxAndroidDemo/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -7,7 +7,8 @@
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
 
     <uses-feature android:name="android.hardware.location.gps"/>
```
Comment (tobrun) [inline MapboxAndroidDemo/src/main/AndroidManifest.xml]: The chromebook mentioned does have a touchscreen but will probably be missing a GPS, 
could we make that optional as well?

**Comment by langsmith** [comment]
@tobrun , last commit has now made GPS optional

**Comment by langsmith** [comment]
@tobrun , do you think this means that we should be making touchscreen and gps optional for all of our apps?

**Comment by tobrun** [comment]
depends on the use-case, eg. a navigation app really needs GPS data.

**Pull request merged**